tsconfig.dev.json

```diff
@@ -1,6 +1,9 @@
 {
 	"extends": "./packages/ts-config/src/tsconfig.json",
 	"compilerOptions": {
-		"target": "ES2020"
+		"target": "ES2020",
+		"noImplicitAny": false,
+		"jsx": "react",
+		"jsxFactory": "EmbedJsx.make"
```

Comment:
I don't think these should be here?

Reply:
cc: @Favna

Reply:
Without these the tests in CI fail because they use this config file @vladfrangu @PyroTechniac